capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java

```diff
@@ -123,6 +116,23 @@ public static String getData(String property, Entity user) {
     return data; 
   }
 
+  public static String getGoalSteps() {
```

Comment:
would it be better to return a list of GoalStep obj? But not sure how this method is going to be used.

Reply:
I was planning on using this returned json in the setJsonGoalSteps of ProgressModel Builder. I didn't want to take away from the normal return types of the DataHandler (everything returned from the file is a json or datastore related entity) so I thought returning the json made sense. However, I could just move the logic from the Builder method to this function.